graph-ir/src/main/scala/wrappers/scala/impl/WOptionsImpl.scala

```diff
@@ -7,11 +7,13 @@ import special.wrappers.OptionWrapSpec
 import scala.collection.compat.immutable.ArraySeq
 
 package impl {
-// Abs -----------------------------------
+  import scalan.reflection.{RClass, RMethod}
+
+  // Abs -----------------------------------
 trait WOptionsDefs extends scalan.Scalan with WOptions {
   self: WrappersModule =>
 
-object WOption extends EntityObject("WOption") {
+class WOptionCls extends EntityObject("WOption") {
```

Comment:
Because Impl is usually when some abstract class/interface is implemented. Here it is to:
- avoid name collision
- have a class type where WOption.type doesn't work (in reflection metadata, see usages of WOptionCls)